src/sdk/src/fragment-components/a-slider.tsx

```diff
@@ -0,0 +1,240 @@
+			labelText='Light'
+			id='light'
+			checked={selectedComponent.light}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					light: checked
+				});
+			}} />
+
+		<TextInput
+			value={selectedComponent.labelText}
+			labelText='Slider label'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				labelText: event.currentTarget.value
+			})} />
+
+		<NumberInput
+			value={selectedComponent.min}
+			label='Min'
+			onChange={(event: any) => {
+                setComponent({
+                    ...selectedComponent,
+                    min: +event.imaginaryTarget.value
+                });
+            }} />
+
+		<NumberInput
+			value={selectedComponent.max}
+			label='Max'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				max: +event.imaginaryTarget.value
+			})} />
+
+		<NumberInput
+			value={selectedComponent.value}
+			label='Value'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				value: +event.imaginaryTarget.value
+			})} />
+
+		<NumberInput
+			value={selectedComponent.step}
+			label='Step'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				step: +event.imaginaryTarget.value
+			})} />
+
+		<NumberInput
+			value={selectedComponent.stepMultiplier}
+			label='Step multiplier'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					stepMultiplier: +event.imaginaryTarget.value
+				});
+			}} />
+
+		<TextInput
+			value={selectedComponent.minLabel}
+			labelText='Min label'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				minLabel: event.currentTarget.value
+			})} />
+
+		<TextInput
+			value={selectedComponent.maxLabel}
+			labelText='Max label'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				maxLabel: event.currentTarget.value
+			})} />
+	</>;
+};
+
+export const ASliderCodeUI = ({ selectedComponent, setComponent }: any) => <TextInput
+	value={selectedComponent.codeContext?.name}
+	labelText='Input name'
+	onChange={(event: any) => {
+		setComponent({
+			...selectedComponent,
+			codeContext: {
+				...selectedComponent.codeContext,
+				name: event.currentTarget.value
+			}
+		});
+	}}
+/>;
+
+export const ASlider = ({
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		rejectDrop={true}
+		{...rest}>
+			<div className={cx(preventCheckEventStyle, componentObj.cssClasses?.map((cc: any) => cc.id).join(' '))}>
+				<Slider
+				id={componentObj.id}
+				labelText={componentObj.labelText}
+				min={componentObj.min}
+				max={componentObj.max}
+				value={componentObj.value}
+				disabled={componentObj.disabled}
+				step={componentObj.step}
+				hideTextInput={componentObj.textInputIsHidden}
+				stepMultiplier={componentObj.stepMultiplier}
+				minLabel={componentObj.minLabel}
+				maxLabel={componentObj.maxLabel}
+				light={componentObj.light} />
```

Comment:
Since there are no children, I believe this should be indented... that's the rule right? @zvonimirfras

Reply:
yep 👍